<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>kotorrent</string>
  </property>
  <widget class="QMenuBar" name="menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionToggle_console"/>
    <addaction name="separator"/>
    <addaction name="actionRotate"/>
    <addaction name="actionRefreshScreen"/>
   </widget>
   <widget class="QMenu" name="menuKobo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Kobo</string>
    </property>
    <addaction name="actionWifi"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuKobo"/>
   <addaction name="menuDebug"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionWifi">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable WI-FI</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="text">
    <string>Rotate</string>
   </property>
  </action>
  <action name="actionRefreshScreen">
   <property name="text">
    <string>Refresh screen</string>
   </property>
  </action>
  <action name="actionToggle_console">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toggle console</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
